test/services/WhatYouOweServiceSpec.scala

```diff
@@ -49,7 +49,12 @@ class WhatYouOweServiceSpec extends TestSupport with FeatureSwitching {
   val mockFinancialDetailsService: FinancialDetailsService = mock[FinancialDetailsService]
   val mockIncomeTaxViewChangeConnector: IncomeTaxViewChangeConnector = mock[IncomeTaxViewChangeConnector]
 
-  object TestWhatYouOweService extends WhatYouOweService(mockFinancialDetailsService, mockIncomeTaxViewChangeConnector)
+  object mockDateService extends DateService() {
+    override def getCurrentDate: LocalDate = LocalDate.parse("2022-04-01")
```

Comment:
Could we use year value of LocalDate.now().getYear instead of hardcoding to 2022?

Reply:
Thank you, Done